crates/oxc_linter/src/utils/vitest.rs

```diff
@@ -19,3 +22,19 @@ pub fn parse_expect_and_typeof_vitest_fn_call<'a>(
         ParsedJestFnCallNew::GeneralJest(_) => None,
     }
 }
+
+pub fn parse_vitest_fn_call<'a>(
+    call_expr: &'a CallExpression<'a>,
+    possible_jest_node: &PossibleJestNode<'a, '_>,
+    ctx: &LintContext<'a>,
+) -> Option<ParsedGeneralJestFnCall<'a>> {
+    let jest_fn_call = parse_jest_fn_call(call_expr, possible_jest_node, ctx)?;
+
+    match jest_fn_call {
+        ParsedJestFnCallNew::GeneralJest(jest_fn_call) => match jest_fn_call.kind {
+            JestFnKind::General(_) => Some(jest_fn_call),
+            _ => None,
+        },
+        _ => None,
+    }
+}
```

Comment:
this feels misleading?

why are `ParsedJestFnCall::Expect` and `ParsedJestFnCall::ExpectTypeOf` not included?

Reply:
ugh i see. github is giving me a poor diff.

i think this change should be reverted, and you should just do:
```
        let Some(ParsedJestFnCallNew::GeneralJest(jest_fn_call)) =
            parse_jest_fn_call(call_expr, possible_jest_node, ctx)
        else {
            return;
        };
```

in the rule/mod.rs

Reply:
done